system/tests/kohana/ORMTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class Kohana_ORMTest extends Unittest_TestCase
+{
+	/**
+	 * Ensure has() doesn't attempt to count non-countables.
+	 *
+	 * @test
+	 * @covers ORM::has
+	 */
+	public function test_has()
+	{
+		$orm = new ORM_Example;
+
+		$result = $orm->has("children", false);
+
+		$this->assertSame(false, $result);
```

Comment:
2. The coding style agreement has been violated:

https://docs.koseven.ga/guide/kohana/conventions#constants

Reply:
Fixed